app/models/weet.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const Weet = sequelize.define(
+    'Weet',
+    {
+      id: {
+        type: DataTypes.INTEGER,
+        autoIncrement: true,
+        primaryKey: true,
+        field: 'id'
+      },
+      content: {
+        type: DataTypes.STRING,
+        allowNull: false,
+        field: 'content'
+      }
+    },
```

Comment:
Don't you need to define the `user_id userId` field here?

Reply:
I thought it was not necessary because I had already defined the association

Reply:
I am not completely sure, but I think it is necessary for creations, updates, gets, etc. If you are sure it works anyway there is no need to add it

Reply:
to create it works, to update and get I have not tried it yet